order/batch.go

```diff
@@ -393,3 +403,42 @@ type BatchStorer interface {
 	// trader to participate in a new batch.
 	MarkBatchComplete() error
 }
+
+// DecrementingBatchIDs lists all possible batch IDs that can exist between a
+// start and end key. Start key must be the larger/higher key, decrementing it a
+// given number of times should finally result in the end key. If the keys
+// aren't related or are too far apart, we return a maximum number of IDs that
+// corresponds to our safety net parameter and the end key won't be contained in
+// the list. The returned list is in descending order, meaning the first entry
+// is the start key, the last entry is the end key.
+func DecrementingBatchIDs(startKey, endKey *btcec.PublicKey) []BatchID {
```

Comment:
not needed anymore?

Reply:
It is, but only in the Prometheus collector on the server side. I think it's still nice to have this on the client side though.